sdk.go

```diff
@@ -192,6 +195,11 @@ func (sdk *SDK) Kubernetes() *k8s.Kubernetes {
 	return k8s.NewKubernetes(sdk.getConn(KubernetesServiceID))
 }
 
+// Translate returns Translate object that is used to operate on Yandex Translate
+func (sdk *SDK) Translate() *translate.Translate {
```

Comment:
I'd prefer to have sdk.AI() here, which contains Translate, Vision and STT (or SpeechToText) subsections.
For other such examples see sdk.MDB().

Reply:
I thought of changing it, but it has a different endpoint: ai-translate

Reply:
It can be done like this:

```go
func (sdk *SDK) AI() *AI {
  return &AI{sdk: sdk} 
}

type AI struct {
  sdk *SDK
}

func (ai *AI) Translate() *translate.Translate { 
  return translate.NewTranslate(ai.sdk.getConn(TranslateServiceID))
}

func (ai *AI) Vision() *vision.Vision {
  return vision.NewVision(ai.sdk.getConn(VisionServiceID))
}
```

Reply:
This looks different than the gen in the sdk.go

Reply:
But still, we have a precedent here: https://github.com/yandex-cloud/go-sdk/blob/master/mdb.go
I guess we'll have to incorporate aggregations like that into the generation.